boards.txt

```diff
@@ -6778,6 +6778,189 @@ adafruit_feather_esp32_v2.menu.DebugLevel.verbose=Verbose
+adafruit_feather_esp32s3.serial.disableRTS=false
+
+adafruit_feather_esp32s3.build.tarch=xtensa
+adafruit_feather_esp32s3.build.bootloader_addr=0x0
+adafruit_feather_esp32s3.build.target=esp32s3
+adafruit_feather_esp32s3.build.mcu=esp32s3
+adafruit_feather_esp32s3.build.core=esp32
+adafruit_feather_esp32s3.build.variant=adafruit_feather_esp32s3_nopsram
+adafruit_feather_esp32s3.build.board=ADAFRUIT_FEATHER_ESP32S3_NOPSRAM
+
+adafruit_feather_esp32s3.build.usb_mode=1
+adafruit_feather_esp32s3.build.cdc_on_boot=1
+adafruit_feather_esp32s3.build.msc_on_boot=0
+adafruit_feather_esp32s3.build.dfu_on_boot=0
+adafruit_feather_esp32s3.build.f_cpu=240000000L
+adafruit_feather_esp32s3.build.flash_size=8MB
+adafruit_feather_esp32s3.build.flash_freq=80m
+adafruit_feather_esp32s3.build.flash_mode=dio
+adafruit_feather_esp32s3.build.boot=qio
+adafruit_feather_esp32s3.build.partitions=default
+adafruit_feather_esp32s3.build.defines=
+adafruit_feather_esp32s3.build.loop_core=-DARDUINO_RUNNING_CORE=1
+adafruit_feather_esp32s3.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+
+adafruit_feather_esp32s3.menu.USBMode.default=USB-OTG
+adafruit_feather_esp32s3.menu.USBMode.default.build.usb_mode=0
+adafruit_feather_esp32s3.menu.USBMode.default.upload.use_1200bps_touch=true
+adafruit_feather_esp32s3.menu.USBMode.default.upload.wait_for_upload_port=true
+adafruit_feather_esp32s3.menu.USBMode.hwcdc=Hardware CDC and JTAG
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.build.usb_mode=1
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.upload.use_1200bps_touch=false
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.upload.wait_for_upload_port=false
+
+adafruit_feather_esp32s3.menu.CDCOnBoot.cdc=Enabled
+adafruit_feather_esp32s3.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+adafruit_feather_esp32s3.menu.CDCOnBoot.default=Disabled
+adafruit_feather_esp32s3.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+adafruit_feather_esp32s3.menu.MSCOnBoot.default=Disabled
+adafruit_feather_esp32s3.menu.MSCOnBoot.default.build.msc_on_boot=0
+adafruit_feather_esp32s3.menu.MSCOnBoot.msc=Enabled
+adafruit_feather_esp32s3.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+adafruit_feather_esp32s3.menu.DFUOnBoot.default=Disabled
+adafruit_feather_esp32s3.menu.DFUOnBoot.default.build.dfu_on_boot=0
+adafruit_feather_esp32s3.menu.DFUOnBoot.dfu=Enabled
+adafruit_feather_esp32s3.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+adafruit_feather_esp32s3.menu.UploadMode.cdc=Internal USB
+adafruit_feather_esp32s3.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+adafruit_feather_esp32s3.menu.UploadMode.cdc.upload.wait_for_upload_port=true
+adafruit_feather_esp32s3.menu.UploadMode.default=UART0
+adafruit_feather_esp32s3.menu.UploadMode.default.upload.use_1200bps_touch=false
+adafruit_feather_esp32s3.menu.UploadMode.default.upload.wait_for_upload_port=false
+
+adafruit_feather_esp32s3.menu.PSRAM.disabled=Disabled
+adafruit_feather_esp32s3.menu.PSRAM.disabled.build.defines=
+adafruit_feather_esp32s3.menu.PSRAM.enabled=Enabled
```

Comment:
No PSRAM, so why Menu entrys PSRAM?

Reply:
mostly to mimic the esp32-s3 dev menu, i can remove if espressif suggests its a good idea!

Reply:
If you don't have any plans to use the ESP32-S3 with PSRAM  on this board, you can remove it to keep it simple for the user. However, this is just a suggestion and does not make any difference to keeping it.

Reply:
It would have been great is all boards would have had their settings predefined, instead of providing menus, but we are very far from such option now :( All menus should be available only on the official chip definition boards, so that they can accommodate more or less every board out there.

Now that said... with S3 things are becoming even more problematic, because we have the OPI flash/psram options. @ladyada as a board manufacturer, have you thought of how you will handle the different module options in your boards? Ex: You are providing a feather with N8R2 modules, but in the future you decide to upgrade to N8R8 (which has OPI PSRAM) or N32R8 (which has OPI Flash and PSRAM), how will you ensure that the user will select the correct option in the menu for their board?

Reply:
ok i amended the PR to remove the flash size selector (8mb only), upload type (usb only) and psram (disabled)

Reply:
we will have a board menu entry for each specific memory configuration - right now we only have the N8 version available :)